src/handlers/jwtToken.go

```diff
@@ -0,0 +1,16 @@
+package handlers
+
+import (
+	"time"
+	"github.com/golang-jwt/jwt/v5"
+)
+
+func GeneratedGitJwtToken(githubAppSecret string, ClientID string) (string, error) {
```

Comment:
looks good, but doesn't work.
no idea why, I will investigate how to convert that python script to go, now I can't get access using this jwt token